Pspcl/Pspcl.Web/Controllers/AccountController.cs

```diff
@@ -53,7 +53,7 @@ public async Task<IActionResult> Login(Pspcl.Models.LoginModel model)
                     await _userManager.UpdateAsync(user);
 
                     _logger.LogInformation($"User '{user.Email}' logged in.");
-                    return RedirectToAction("Index", "Home");
+                    return RedirectToAction("Index", "Dashboard");
```

Comment:
You can keep Dashboard action under HomeController, no need to create a separate new controller for a single Dashboard action.